packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -205,6 +205,13 @@ function Thread( {
 			aria-label={ ariaLabel }
 			aria-expanded={ isSelected }
 		>
+			<Button
+				className="editor-collab-sidebar-panel__skip-link"
+				variant="secondary"
+				__next40pxDefaultSize
```

Comment:
```suggestion
				size="compact"
```

40px might be a little big